pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py

```diff
@@ -0,0 +1,187 @@
+"""rename tables to plurals 
+
+Revision ID: b6e28faea978
+Revises: 90afcc0d6d9f
+Create Date: 2021-01-25 21:50:58.043768
+
+"""
+from alembic import op
+from sqlalchemy import create_engine
+from sqlalchemy.engine import reflection
+from sqlalchemy import MetaData
+import re
+
+# revision identifiers, used by Alembic.
+from sqlalchemy.engine.reflection import Inspector
+
+VERSION = '_version'
+
+# revision identifiers, used by Alembic.
+revision = 'b6e28faea978'
+down_revision = '90afcc0d6d9f'
+branch_labels = None
+depends_on = None
+
+###
+# get_pk_constraint
+
+table_mapping = {'bcol_payment_account ': 'bcol_payment_accounts',
+                 'cfs_account': 'cfs_accounts',
+                 'cfs_account_status_code': 'cfs_account_status_codes',
+                 'corp_type': 'corp_types',
+                 'credit_payment_account': 'credit_payment_accounts',
+                 'daily_payment_batch': 'daily_payment_batches',
+                 'daily_payment_batch_link': 'daily_payment_batch_links',
+                 'distribution_code': 'distribution_codes',
+                 'distribution_code_link': 'distribution_code_links',
+                 'ejv_batch': 'ejv_batches',
+                 'ejv_batch_link': 'ejv_batch_links',
+                 'error_code': 'error_codes',
+                 'fee_code': 'fee_codes',
+                 'fee_schedule': 'fee_schedules',
+                 'filing_type': 'filing_types',
+                 'internal_payment_account': 'internal_payment_accounts',
+                 'invoice': 'invoices',
+                 'invoice_batch': 'invoice_batches',
+                 'invoice_batch_link': 'invoice_batch_links',
+                 'invoice_reference': 'invoice_references',
+                 'invoice_reference_status_code': 'invoice_reference_status_codes',
+                 'invoice_status_code': 'invoice_status_codes',
+                 'line_item_status_code': 'line_item_status_codes',
+                 'notification_status_code': 'notification_status_codes',
+                 'payment': 'payments',
+                 'payment_account': 'payment_accounts',
+                 'payment_line_item': 'payment_line_items',
+                 'payment_method': 'payment_methods',
+                 'payment_status_code': 'payment_status_codes',
+                 'payment_system': 'payment_systems',
+                 'payment_transaction': 'payment_transactions',
+                 'receipt': 'receipts',
+                 'statement': 'statements',
+                 'transaction': 'transactions',
```

Comment:
transaction table I think is created by continuum, so we cannot rename them ?

Reply:
will remove it.